### PR TITLE
search by numeric

### PR DESCRIPTION
**Pull request by kassemitani**
Added ability to search by numeric as i need it in my case

### CODE REVIEW

File: IsoCountryCodes.swift [hunk 1]
```diff
@@ -19,6 +19,12 @@ class IsoCountryCodes {
         return (!country.isEmpty) ? country[0] : IsoCountryInfo(name: "", numeric: "", alpha2: "", alpha3: "", calling: "", currency: "", continent: "")
     }
     
+    class func searchByNumeric( numeric:String ) -> IsoCountryInfo {
+        var country = IsoCountries.allCountries.filter( { $0.numeric == numeric } )
```
Comment (funky-monkey) [inline IsoCountryCodes.swift]: Can you make this a `let`? It is never mutated.

Comment (funky-monkey) [inline IsoCountryCodes.swift]: Can you change this to a `let`? If this is changed i'll merge it :)

**Review by funky-monkey — CHANGES_REQUESTED** [review]
For the rest it looks good, just a small comment

File: IsoCountryCodes.swift (hunk 1 above)
Comment (kassemitani) [reply IsoCountryCodes.swift]: done

**Pull request merged**